### PR TITLE
many small changes from going over all props

### PR DESCRIPTION
**Pull request by tgberkeley**
### All Submissions:

- [ ] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/reflex-dev/reflex/blob/main/CONTRIBUTING.md) file?
- [ ] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/reflex-dev/reflex/pulls ) for the desired changed?

### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### New Feature Submission:

- [ ] Does your submission pass the tests? 
- [ ] Have you linted your code locally prior to submission?

### Changes To Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### **After** these steps, you're ready to open a pull request.

    a. Give a descriptive title to your PR.

    b. Describe your changes.

    c. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: reflex/components/radix/themes/components/alertdialog.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ class AlertDialogContent(el.Div, RadixThemesComponent):
     tag = "AlertDialog.Content"
 
     # The size of the content.
-    size: Var[LiteralSize]
+    size: Var[Literal["1", "2", "3", "4"]]
```
Comment (masenf) [inline reflex/components/radix/themes/components/alertdialog.py:45]:
```suggestion
    size: Var[LiteralContentSize]
```

Does that not work?

File: reflex/components/radix/themes/components/dropdownmenu.py [hunk 2]
```diff
@@ -44,6 +44,18 @@ class DropdownMenuContent(RadixThemesComponent):
 
     tag = "DropdownMenu.Content"
 
+    # Button size "1" - "4"
```
Comment (masenf) [inline reflex/components/radix/themes/components/dropdownmenu.py:47]:
```suggestion
    # Dropdown menu size "1" - "2"
```

File: reflex/components/radix/themes/components/dropdownmenu.py [hunk 3]
```diff
@@ -44,6 +44,18 @@ class DropdownMenuContent(RadixThemesComponent):
...
+    size: Var[Literal["1", "2"]]
+
+    # Variant of button: "solid" | "soft" | "outline" | "ghost"
+    variant: Var[Literal["solid", "soft"]]
+
+    # Override theme color for button
+    color_scheme: Var[LiteralAccentColor]
+
+    # Whether to render the button with higher contrast color against background
+    high_contrast: Var[bool]
```
Comment (masenf) [inline reflex/components/radix/themes/components/dropdownmenu.py:57]:
```suggestion
    # Variant of dropdown menu: "solid" | "soft"
    variant: Var[Literal["solid", "soft"]]

    # Override theme color for dropdown menu
    color_scheme: Var[LiteralAccentColor]

    # Whether to render the dropdown menu with higher contrast color against background
    high_contrast: Var[bool]
```

**Review by masenf — CHANGES_REQUESTED** [review]

File: reflex/components/radix/themes/components/alertdialog.py (hunk 1 above)
Comment (tgberkeley) [reply reflex/components/radix/themes/components/alertdialog.py:45]: changed

File: reflex/components/radix/themes/components/dropdownmenu.py (hunk 2 above)
Comment (tgberkeley) [reply reflex/components/radix/themes/components/dropdownmenu.py:47]: changed

File: reflex/components/radix/themes/components/dropdownmenu.py (hunk 3 above)
Comment (tgberkeley) [reply reflex/components/radix/themes/components/dropdownmenu.py:57]: changed

**Review by masenf — CHANGES_REQUESTED** [review]
regen pyi and we can take it

**Review by picklelo — APPROVED** [review]

**Pull request merged**